ViAn/GUI/mainwindow.ui

```diff
@@ -412,27 +412,27 @@
      <property name="title">
       <string>Shape</string>
      </property>
-     <addaction name="actionRectangle"/>
-     <addaction name="actionCircle"/>
-     <addaction name="actionLine"/>
-     <addaction name="actionArrow"/>
-     <addaction name="actionPen"/>
-     <addaction name="actionText"/>
+     <addaction name="action_rectangle"/>
+     <addaction name="action_circle"/>
+     <addaction name="action_line"/>
+     <addaction name="action_arrow"/>
+     <addaction name="action_pen"/>
+     <addaction name="action_text"/>
     </widget>
-    <addaction name="actionColour"/>
+    <addaction name="action_colour"/>
     <addaction name="menuShape"/>
-    <addaction name="actionUndo"/>
-    <addaction name="actionClear"/>
-    <addaction name="actionContrast_Brightness"/>
+    <addaction name="action_undo"/>
+    <addaction name="action_clear"/>
+    <addaction name="action_contrast_brightness"/>
    </widget>
    <addaction name="menuFile"/>
    <addaction name="menuEdit"/>
    <addaction name="menuView"/>
    <addaction name="menuTools"/>
    <addaction name="menuHelp"/>
```

Comment:
Shouldn't all these be fixed as well?
menuShape, menuFile, menuEdit, menuView, menuTools, menuHelp